best_practices/configuration.rst

```diff
@@ -152,10 +152,10 @@ valori, mentre le stesse classi non posso accedere ai parametri del contenitore.
         }
     }
 
-L'unico svantaggio da considerare, nell'utilizzo delle costanti come opzioni di configurazione, è
+L'unico svantaggio da considerare nell'utilizzo delle costanti come opzioni di configurazione, è
 che non possono essere ridefinite facilmente nei test.
 
-Non usare la configurazione semantica
+Configurazione semantica: non usarla
 -------------------------------------
```

Comment:
Con un carattere in meno nel titolo, serve un carattere in meno nella sottilineatura